Makefile.am

```diff
@@ -219,27 +219,24 @@ libraft_la_CFLAGS = $(AM_CFLAGS)
 libraft_la_LDFLAGS = $(UV_LIBS)
 
 raft_core_unit_test_SOURCES = \
-  src/tracing.c \
-  src/raft/byte.c \
-  src/raft/compress.c \
-  src/raft/configuration.c \
-  src/raft/err.c \
-  src/raft/flags.c \
-  src/raft/heap.c \
-  src/raft/log.c \
+  $(basic_dqlite_sources) \
```

Comment:
I'm a bit concerned about making the raft test artifacts link in arbitrary dqlite object code or even libdqlite itself---feels like we should maintain a separation here even if it requires compiling a subset of source files (like tracing.c) more than once.

Reply:
The problem is that we are going to be using code from dqlite in Raft soon. We need to read pages, check the wal, etc. Right now I needed for the state machines but in the future I thought that we were going to combine both Raft and dqlite.